source/features/dim-bots.tsx

```diff
@@ -20,8 +20,10 @@ const commitSelectors = botNames.map(bot => `.commit-author[href$="?author=${bot
 commitSelectors.push('.commit-author[href$="%5Bbot%5D"]'); // Generic `[bot]` label in author name
 const commitSelector = commitSelectors.join(',');
 
-const prSelectors = botNames.map(bot => `.opened-by [title="pull requests opened by ${bot}"]`);
+const prSelectors = [];
 prSelectors.push(
+	...botNames.map(bot => `.opened-by [title*="pull requests opened by ${bot}"]`),
```

Comment:
I kept it because it was there but I'm not sure where it is used (or not anymore?)

Reply:
I presume that the copy was just updated and it can now be replaced instead of extended